content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -353,6 +363,22 @@ In addition to ECS task tags, the following out-of-the-box tags are applied to c
 | `is_aws_ecs_on_ec2`     | All EC2 compute costs associated with running ECS on EC2. |
 | `is_aws_ecs_on_fargate` | All costs associated with running ECS on Fargate. |
 
+### Data transfer
+
+The following non-exhaustive list of out-of-the-box tags are applied to cost metrics associated with Kubernetes workloads:
```

Comment:
Is there somewhere customers can go to see an exhaustive list? Like, "For a full list of tags that are applied, see X"

```suggestion
The following list of out-of-the-box tags are applied to cost metrics associated with Kubernetes workloads:
```

Reply:
There isn't, because the exhaustive list is dependent on custom tags that the customer may have created on their side. Omitting "non-exhaustive" sounds good to me.